src/java.desktop/windows/classes/sun/awt/windows/WClipboard.java

```diff
@@ -109,13 +115,32 @@ protected void clearNativeContext() {}
      * @throws IllegalStateException if the clipboard has not been opened
      */
     @Override
-    public native void openClipboard(SunClipboard newOwner) throws IllegalStateException;
-    /**
```

Comment:
was this comment accidentally deleted?

Reply:
Restored the comment and adjusted the comment on `openClipboard` to better address the required calling sequence.